apm-sniffer/optional-plugins/trace-ignore-plugin/src/main/java/org/apache/skywalking/apm/plugin/trace/ignore/matcher/FastPathMatcher.java

```diff
@@ -83,6 +90,10 @@ private boolean normalMatch(String pat, int p, String str, int s) {
 
     private boolean wildcardMatch(String pat, int p, String str, int s) {
         char pc = safeCharAt(pat, p);
+        //if pat already arrival end,and str in position of s is not '/',then return true
+        if (pc == '\u0000' && safeCharAt(str, s) != '/') {
+            return true;
```

Comment:
I think this change will cause a bug that 'abc/*' matching 'abc/foo/bar'

Reply:
i review it